{"name": "test_vwo_login_positive", "status": "broken", "statusDetails": {"message": "AttributeError: 'TestLogin' object has no attribute 'user'", "trace": "self = <tests.vmoLoginTests.pom.test_login.TestLogin object at 0x1073591d0>\nsetup = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"12dfce369c36f64d8b8b2cbf07cb526c\")>\n\n    @allure.epic(\"VWO Login Test\")\n    @allure.feature(\"TC#1 - VWO App Positive Test\")\n    @pytest.mark.usefixtures(\"setup\")\n    def test_vwo_login_positive(self, setup):\n        driver = setup\n        driver.get(self.base_url)\n        loginPage = LoginPage(driver)\n>       loginPage.login_to_vwo(user=self.user, pwd=self.pwd)\nE       AttributeError: 'TestLogin' object has no attribute 'user'\n\ntest_login.py:34: AttributeError"}, "start": 1708231319646, "stop": 1708231326528, "uuid": "0c88407c-b080-43f4-ba30-f3720c771bac", "historyId": "063967906f3f3e8bd67115b608b7b12e", "testCaseId": "063967906f3f3e8bd67115b608b7b12e", "fullName": "tests.vmoLoginTests.pom.test_login.TestLogin#test_vwo_login_positive", "labels": [{"name": "feature", "value": "TC#1 - VWO App Positive Test"}, {"name": "epic", "value": "VWO Login Test"}, {"name": "tag", "value": "@pytest.mark.usefixtures('setup')"}, {"name": "parentSuite", "value": "tests.vmoLoginTests.pom"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "GaneshGrandhi.local"}, {"name": "thread", "value": "14040-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.vmoLoginTests.pom.test_login"}]}